tests/backup_helper.go

```diff
@@ -8752,3 +8752,59 @@ func ValidatePVCCleanup(pvcBefore, pvcAfter []string) error {
 
 	return nil
 }
+
+type MaintenanceJobType string
+
+const (
+	FullMaintenanceJob  MaintenanceJobType = "full-maintenance-repo"
+	QuickMaintenanceJob MaintenanceJobType = "quick-maintenance-repo"
+)
+
+// ModifyMaintenanceJobFrequency modifies the frequency of the cron job
+func ModifyMaintenanceJobFrequency(backupLocationName string, frequencyInMinutes int, maintenanceJobType MaintenanceJobType) error {
```

Comment:
How about having complete cron in parameter , considering the hourly schedules.

Reply:
If you want hourly schedule we need to give it in minutes. I did this to reduce the input parameters. Ideally we would want this to be in minutes.